illwill.nim

```diff
@@ -542,13 +562,17 @@ proc illwillDeinit*() =
   resetAttributes()
   showCursor()
 
-proc getKey*(): Key =
+proc getKey*(blocking: bool = false): Key =
   ## Reads the next keystroke in a non-blocking manner. If there are no
```

Comment:
Update method comment and the general documentation everywhere.

Reply:
@johnnovak ok, i will update the relate comments when i am free :)

Reply:
Ok, saw my other comment too?

> Could be handy but I don't like the implementation. getKey should just use getch from the terminal module internally instead.

Reply:
@johnnovak  yeah, i got it